libraries/node-core-library/src/LegacyAdapters.ts

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+import { sort as timsort } from 'timsort';
```

Comment:
It'd be cool if we could somehow only install this dependency on Node versions below 11.

Reply:
I don't know of any easy way to do that. (?)

BTW `timsort` itself is tiny and has no dependencies.

Reply:
Probably not worth worrying about then.